sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -51,6 +51,8 @@ class InteractiveBrowserCredential(InteractiveCredential):
         will cache tokens in memory.
     :paramtype cache_persistence_options: ~azure.identity.TokenCachePersistenceOptions
     :keyword int timeout: seconds to wait for the user to complete authentication. Defaults to 300 (5 minutes).
+    :keyword bool allow_broker: Brokers provide single sign-on, device identification, and application identification
+        verification. If this parameter is set to True, the broker will be used when possible. Defaults to False.
```

Comment:
FYI: MSAL teams are currently under discussion on whether this parameter shall be changed to be platform specific, meaning downstream callers would need to opt in only on supported platform(s). Azure Identity shall mark this release a beta release, until we settle down on one way or another.